src/Maestro/ReleasePipelineRunner/ReleasePipelineRunner.cs

```diff
@@ -371,7 +373,7 @@ private async Task CreateGitHubIssueAsync(int buildId, int releaseId, string rel
                     Logger.LogInformation($"Going to include {author} in the created issue");
 
                     string title = $"Release '{releaseName}' with id {releaseId} failed";
-                    string description = $"Something failed while trying to publish artifacts for build [{buildId}]({build.Id})." +
+                    string description = $"Something failed while trying to publish artifacts for build [{build.AzureDevOpsBuildNumber}](https://dnceng.visualstudio.com/internal/_build/results?buildId={build.AzureDevOpsBuildId})." +
```

Comment:
I think we can make this message a bit more generic as we are doing more than publishing now :-)

Reply:
True!